recipe/meta.yaml

```diff
@@ -36,10 +36,25 @@ requirements:
     - traitlets
 
 test:
+  requires:
+    - nose
+    - setuptools
+    - wheel
+    - nose
+    - coverage
+    - nodejs
+    - phantomjs
```

Comment:
> Conda-forge isn't the CI for the project, though.

@pelson: are you 👍  to increasing test robustness, or not using CF as a CI?

I would maintain it probably does make sense as an integration testing platform, as we would be testing the core/extensions _as installed_, rather than in a more-bespoke, per-project test environment.

> If you want to help tackle the robustness of the notebook js test suite, that would be terrific.

This goes back to the discussion at the dev meeting: moving the capability of "test the notebook backend and frontend (by itself | with an extension | with a mess of extensions)" out of notebook into something easy to install would help.

If we're going to put some effort into this, we should seriously consider:
- staying with casper (but upgrade to casper 1.1.1 and phantom2)
  - this is already done... everything just appears to work here with these deps upgraded, so that might be an easy place to start
  - getting coverage with casper is non-trivial, but is something I would like to explore
- going with something else in the same vein, but newer (nightmare)
- paying the effort tax, and go with native browsers and selenium+jdk on the test platforms
  - appears to be mostly adopted on jupyter-js-notebook, but uses mocks instead of the real backend...
  - good, as it would be more reproducible
- paying the saucelabs tax (another account to manage)
 - best browser coverage

I guess near-term it's worth upgrading to the newest casper/phantom and see if that helps with some of our stability issues. I like having something with the ability to take screenshots, even if we are not capturing image changes over time.